make/command

```diff
@@ -12,3 +12,8 @@ bin/print$(EXE) : bin/cmdstan/print.o
 bin/stansummary$(EXE) : bin/cmdstan/stansummary.o
 	@mkdir -p $(dir $@)
 	$(LINK.c) -O$(O_STANC) $(OUTPUT_OPTION) $< $(LDLIBS)
+
+.PRECIOUS: bin/diagnose$(EXE)
```

Comment:
You'll want to add the `bin/diagnose` target to the `build` target defined in `/makefile`

You'll also want to add it to the model build step in: `make/models`.

We might be able to simplify the makefiles somehow, but that's for a different PR.